src/modules/75-cf/components/FlagArchiving/CannotArchiveWarning.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FC, useMemo } from 'react'
+import { Layout, TableV2, Text } from '@harness/uicore'
+import { FontVariation } from '@harness/design-system'
+import { Link, useParams } from 'react-router-dom'
+import type { CellProps, Column } from 'react-table'
+import useActiveEnvironment from '@cf/hooks/useActiveEnvironment'
+import routes from '@common/RouteDefinitions'
+import { useStrings } from 'framework/strings'
+import type { Feature, Prerequisite } from 'services/cf'
+
+export interface CannotArchiveWarningProps {
+  flagData: Feature
+}
+
+const CannotArchiveWarning: FC<CannotArchiveWarningProps> = ({ flagData }) => {
+  const { getString } = useStrings()
+  const { withActiveEnvironment } = useActiveEnvironment()
+  const { projectIdentifier, orgIdentifier, accountId } = useParams<Record<string, string>>()
+
+  const columns: Column<Prerequisite>[] = useMemo(
+    () => [
+      {
+        Header: '',
+        id: 'flagDetails',
+        Cell: ({ row }: CellProps<Prerequisite>) => {
+          return (
+            <Layout.Horizontal>
+              <Text padding={{ right: 'xsmall' }}>{getString('cf.environments.environmentID')}</Text>
```

Comment:
Probably best to just use `idLabel` instead of getting a string from another page.
```suggestion
              <Text padding={{ right: 'xsmall' }}>{getString('idLabel')}</Text>
```

Reply:
I've selected one from the `common` string yaml